toolchain/binutils-2.43-01-i386-pc-phoenix.patch

```diff
@@ -0,0 +1,80 @@
+diff --git a/bfd/config.bfd b/bfd/config.bfd
+index 6553aac1e99..596867b1700 100644
+--- a/bfd/config.bfd
++++ b/bfd/config.bfd
+@@ -783,6 +783,10 @@ case "${targ}" in
+     targ_defvec=i386_elf32_vxworks_vec
+     targ_underscore=yes
+     ;;
++  i[3-7]86-*-phoenix*)
++    targ_defvec=i386_elf32_vec
++    targ_selfvecs=
++    ;;
+ 
+   ia16-*-elf)
+     targ_defvec=i386_elf32_vec
+diff --git a/gas/configure.tgt b/gas/configure.tgt
+index d58f21873a3..0ce4759efda 100644
+--- a/gas/configure.tgt
++++ b/gas/configure.tgt
+@@ -267,6 +267,7 @@ case ${generic_target} in
+   i386-*-*nt*)				fmt=coff em=pe ;;
+   i386-*-rdos*)				fmt=elf ;;
+   i386-*-darwin*)			fmt=macho ;;
++  i386-*-phoenix*)			fmt=elf ;;
+ 
+   ia16-*-elf*)				fmt=elf ;;
+ 
+diff --git a/ld/Makefile.am b/ld/Makefile.am
+index 6a9833e5775..99ec9f7af3d 100644
+--- a/ld/Makefile.am
++++ b/ld/Makefile.am
```

Comment:
Compared to the old patch there are no
```+@AMDEP_TRUE@@am__include@ @am__quote@./$(DEPDIR)/eelf_i386_phoenix.Pc@am__quote@```
lines in Makefile.am and Makefile.in. Is this intended?

Reply:
Yes, they reworked the build systems a little bit and those lines are no longer needed